src/OpenTelemetry.Exporter.ZPages/README.md

```diff
@@ -1,26 +0,0 @@
-# ZPages Exporter for OpenTelemetry .NET
```

Comment:
maybe keep this page for sometime, with a description saying this is decommissioned. Or we think a nuget.org deprecation is fine?

Reply:
Both the package and the readme file have very low number of visitors so I don't feel there is a need to keep the readme.